src/SCRIPTS/BF/PAGES/rates.lua

```diff
@@ -75,7 +75,7 @@ if apiVersion >= 1.016 then
     end
 end
 
-if apiVersion >= 1.016 then
+if apiVersion >= 1.016 and apiVersion <= 10.44 then
```

Comment:
Should be 1.044. We should change the api version calculation to get rid of the zero before minor version, but that's for another PR 🙂

Reply:
Fixed. Yes good call as this was programmer blindness ;-)